client-v2/src/actions/__tests__/ArticleFragments.spec.ts

```diff
@@ -77,25 +85,36 @@ const buildStore = (added: ArticleFragmentSpec) => {
   return createStore(root, state as any, applyMiddleware(thunk));
 };
 
-const insert = (
+const insert = async (
   [uuid, id]: [string, string],
```

Comment:
what do the uuid and id represent here?

Reply:
They are used to build a stub article fragment seen as in these test we're just testing the way `uuid`s and `id`s affect things like deduplication / collection caps etc. It's basically a short hand for creating article fragments.

Reply:
Ah I get what you mean, I'll add a variable name and destructure inside the function.